Jenkinsfile

```diff
@@ -328,6 +334,7 @@ pipeline {
             stage('Notify') {
               options { skipDefaultCheckout() }
               steps {
+                setEnvVar('PRE_RELEASE_STAGE', 'false')
```

Comment:
This flag will ensure the top-level post failure won't trigger again another notification.